db/schema.rb

```diff
@@ -211,8 +211,6 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "relationship"
-    t.decimal "monthly_income"
-    t.boolean "has_income"
```

Comment:
these 2 lines seem to have sneaked in from another PR

Reply:
Actually, these 2 were duplicated by this PR: https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/667/files#diff-1acd2e7e27a227829d5d14a91c863bb6R219
So it's good to remove them.